model/Core/Classes/PackageVerificationCode.md

```diff
@@ -18,19 +18,21 @@ This identifier enables a recipient to determine if any file in the original pac
 
 Algorithm:
 
+    hashValue = 0
     templist = empty list of strings
 
-    for all files in the package
-        if file is an "excludes" file
+    for all files in the package {
+        if file is a packageVerificationCodeExcludedFile
             skip it /* exclude SPDX analysis file(s) */
         else
-            append algorithm(contents of file) to templist
+            append "algorithm(file)/n" to templist
```

Comment:
No, it's a list, not a string.
It has different elements, you don't need "separators".

Reply:
The byte specific implementation of list elements (of strings) varies depending on underlying language (JAVA, Python) as well it may have be impacted by the platform (linux, macOS,windows)  the library is implemented on.  

The goal here was to  is to be compatible with 2.3 - not to create a "better" algorithm.